src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -365,6 +365,24 @@ public class FormattingData
         /// </summary>
         public string Underline { get; } = "\x1b[4m";
 
+        /// <summary>
+        /// Gets value to turn off strikethrough.
+        /// </summary>
+        public string StrikethroughOff { get; } = "\x1b[29m";
+
+        /// <summary>
+        /// Gets value to turn on strikethrough.
+        /// </summary>
+        public string Strikethrough { get; } = "\x1b[9m";
+
+        /// <summary>
+        /// Gets ANSI representation of a hyperlink.
+        /// </summary>
+        public string ToHyperlink(string text, Uri link)
```

Comment:
`To` implies that the `$PSStyle` object can be turned into a hyperlink (like `DateTime.ToWhateverString()`)... so I don't quite like `To` over `Format`. Maybe even:

```
$PSStyle.Hyperlink(...)
```

is fine. I just added the Format to differenciate between a method and prop.

Reply:
I noticed you have `FromRgb`... could do `FromUri`?

Reply:
I see.  I'll change it to Format and wait for user feedback.